mslib/msui/msui_mainwindow.py

```diff
@@ -1027,7 +1040,7 @@ def create_view_handler(self, _type):
                 show_popup(self.mscolab.ui, "Error", "Session expired, new login required")
 
     def restore_views_for_active_flighttrack(self):
-        if not self.active_flight_track:
+        if not self.active_flight_track or self.mscolab.active_op_id:
```

Comment:
```suggestion
        if not (self.active_flight_track or self.mscolab.active_op_id):
```

Otherwise this means "if there is no active flight track or if there is a active op id", which doesn't seem right.